### PR TITLE
ebs-br: change command line flags of restore

### LINKED ISSUE #38011: br: parameter rename for ebs snapshot backup and restore
## Bug Report
current implement for ebs snapshot backup restore for tide-operator
```
br backup full --type=aws-ebs/gcp-ps // backup from various snapshot
br restore full --meta-phase // restore metadata
br restore full --data-phase // restore cluster data
```

suggest change to 
```
br backup full --type=aws-ebs/gcp-ps // backup from various snapshot
br restore full --type=aws-ebs --prepare // i.e. current phase one
br restore full --type=aws-ebs // restore cluster data
```

we would like to remove phase concepts for  the feature
Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

kubectl logs backup_pod

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: close #38011

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by fengou1 — APPROVED** [review]
/LGTM

File: br/pkg/task/restore.go [hunk 1]
```diff
@@ -171,15 +171,15 @@ type RestoreConfig struct {
...
-	MetaPhase           bool                  `json:"meta-phase"`
-	DataPhase           bool                  `json:"data-phase"`
-	OutputFile          string                `json:"output-file"`
-	SkipAWS             bool                  `json:"skip-aws"`
-	CloudAPIConcurrency uint                  `json:"cloud-api-concurrency"`
-	VolumeType          pconfig.EBSVolumeType `json:"volume-type"`
-	VolumeIOPS          int64                 `json:"volume-iops"`
-	VolumeThroughput    int64                 `json:"volume-throughput"`
-	ProgressFile        string                `json:"progress-file"`
+	FullBackupType      FullBackupType        `json:"full-backup-type" toml:"full-backup-type"`
```
Comment (3pointer) [inline br/pkg/task/restore.go:174]: should it be type?

**Pull request merged**